<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceValorantBGVideoClass</class>
 <widget class="QMainWindow" name="ReplaceValorantBGVideoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplaceValorantBGVideo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ReplaceValorantBGVideo.qrc">
    <normaloff>:/ReplaceValorantBGVideo/Icon1.ico</normaloff>:/ReplaceValorantBGVideo/Icon1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="input_gamedir">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>761</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_replace">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>被替换：</string>
    </property>
   </widget>
   <widget class="QToolButton" name="select_gamedir">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_selectVideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>选择视频：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_videodir">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>761</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="select_video">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Folder">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AutoDetect">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>自动检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_videoList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视频列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VList_Add">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>加入</string>
    </property>
   </widget>
   <widget class="QWidget" name="VideoReview" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>320</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(17, 17, 17);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Preview_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>预览替换视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Preview_0">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>预览被替换视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VList_Del">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QListWidget" name="videoList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stopPreview">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_import">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_showDW">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DebugWindow</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ReplaceValorantBGVideo.qrc"/>
 </resources>
 <connections/>
</ui>
